internal/blockchain/validate.go

```diff
@@ -1022,6 +1030,27 @@ func (b *BlockChain) checkBlockHeaderPositional(header *wire.BlockHeader, prevNo
 			str = fmt.Sprintf(str, header.Timestamp, medianTime)
 			return ruleError(ErrTimeTooOld, str)
 		}
+
+		// A block on the test network must have a timestamp that is at least
+		// one minute after the previous one once the maximum allowed difficulty
+		// has been reached.  This helps throttle ASICs and GPUs since it's not
+		// reasonable to require high-powered hardware to keep the test network
+		// running smoothly.
+		//
+		// This rule is only active on the version 3 test network once the max
+		// diff activation height has been reached.
+		blockHeight := prevNode.height + 1
+		if b.minTestNetTarget != nil &&
+			expDiff <= standalone.BigToCompact(b.minTestNetTarget) &&
+			(!b.isTestNet3() || blockHeight >= testNet3MaxDiffActivationHeight) {
+
+			minTime := time.Unix(prevNode.timestamp, 0).Add(time.Minute)
+			if !header.Timestamp.After(minTime) {
```

Comment:
This would seem to error on exactly one minute, but that's at odds with the comment.

Reply:
No sorry, I'm reading this wrong

Reply:
Your initial comment is correct in that it is rejecting the boundary condition (exactly one minute).  My thought here was that it should reject that case because timestamps are required to be at least one second after the median time of the previous blocks, so rejecting the boundary condition would retain that effectively making it have to be 1 minutes after that minimum possible time that it could have been otherwise.

It's fair to note that the comment is a bit ambiguous here and can easily read as "1 minute inclusive".

We can either change the check to allow the exact boundary condition (e.g. `if header.Timestamp.Before(minTime) { /* reject */ }` or update the comment to better call out that it's 1 minute and 1 second.

Reply:
No feeling either way.  I just felt a disconnect between the code and comment.

Reply:
After considering it a bit, I went ahead and updated the check itself to allow exactly one minute.  I think having it set to exactly one minute is an easier to understand (and thus reason about) rule than "1 minute after what would have been allowed otherwise".

Updated.